### PR TITLE
fix: replace imgur with s3 for Chinese guide without conflict

### PR DESCRIPTION
**Pull request by ahmaxed**
(cherry picked from commit 21e3afaee0f23d700f76ea662bc193b392fc54ac)

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Reference #36007

### CODE REVIEW

File: guide/chinese/cplusplus/index.md [hunk 1]
```diff
@@ -78,8 +79,8 @@ int main()
 *   不要忘记告诉计算机这是main（）函数的结束。要执行此操作，请添加结束大括号“}”。如果不包含**}，**则在程序执行前会遇到错误。
 
 ### 代码看起来应该是这样的：
-
-![图](https://i.imgur.com/d1liGwI.png)
+https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/chinese/cplusplus/index.md]: Is this line needed?
```suggestion
```

File: guide/chinese/cplusplus/tokens-variables/index.md [hunk 2]
```diff
@@ -176,8 +177,8 @@ int 9a;
 如果未初始化变量，则它包含垃圾值。例如：
 
 所以就盒子而言，你可以把它想象成 -
-
-![图](https://i.imgur.com/YdbgWHL.png)
+https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/chinese/cplusplus/tokens-variables/index.md]: Is this line needed?
```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@ahmadabdolsaheb See my requested changes.

**Comment by ahmadabdolsaheb** [comment]
@RandellDawson, they should be resolved now.

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb OK, this PR looks good now.

**Review by RandellDawson — APPROVED** [review]

**Comment by ahmadabdolsaheb** [comment]
I double checked this. it should be good to go.

**Comment by ahmadabdolsaheb** [comment]
@thecodingaviator, I would appreciate it If you could take a look into this.

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**